mne/viz/utils.py

```diff
@@ -924,11 +924,12 @@ def plot_sensors(info, kind='topomap', ch_type=None, title=None,
     show_names : bool | array of str
         Whether to display all channel names. If an array, only the channel
         names in the array are shown. Defaults to False.
-    ch_groups : 'position' | array of shape (n_ch_groups, n_picks) | None
+    ch_groups : 'position' | array of shape (n_ch_groups, n_picks) | array_like | None
```

Comment:
```suggestion
    ch_groups : 'position' | list of lists | None
```
I think we can talk about it this way since it's the most natural way to deal with the data

Reply:
I agree that list of lists is the most natural way to specify channel groups. It actually turns out that the list of list functionality is already tested in `test_plot_sensors.py` by the line `raw.plot_sensors(ch_groups=[[0, 1, 2], [3, 4]])` . 

Perhaps I could add a test to show that the function works with numpy arrays and numpy object arrays as well?

Reply:
> Perhaps I could add a test to show that the function works with numpy arrays and numpy object arrays as well?

Yes I guess you could. But if it emits a warning on a ragged array then maybe it's not worth it...

Reply:
I believe my first commit fixed both the warning and the error I mentioned in the original issue - they were a result of insufficient type-checking within the function. I went ahead and added the tests.